.babelrc

```diff
@@ -1,6 +1,10 @@
 {
   "presets": ["react", ["env", {"modules": false}]],
-  "plugins": ["babel-plugin-transform-exponentiation-operator", "transform-class-properties"],
+  "plugins": [
+    "babel-plugin-transform-exponentiation-operator",
+    "transform-class-properties",
+    ["transform-object-rest-spread", { "useBuiltIns": true }],
```

Comment:
Currently, we are using `babel-preset-env` which include all the features that are part of the standard Ecmascript. Even though [object rest spread is in stage 3](https://github.com/tc39/proposal-object-rest-spread), it is not part of the standard yet and has some [known issues](https://github.com/tc39/proposal-object-rest-spread/blob/master/Issues.md). Do we want to write non-standard JS? I am more tempted to see the progress of this feature and decide to adopt it later on when it reach stage 4. In that way, we will avoid re writing code.

Reply:
https://github.com/tc39/proposals/commit/7a04292f2e6f11fbcfef8a7b20a0bf5aedba79a4 🎉

Reply:
My bad, I didn't check [finished-proposals](https://github.com/tc39/proposals/blob/master/finished-proposals.md), I didn't see it in the Readme of tc39/proposals and I assumed the worst. In that case 👍